railties/lib/rails/tasks/framework.rake

```diff
@@ -47,8 +47,9 @@ namespace :app do
         @app_generator ||= begin
           require 'rails/generators'
           require 'rails/generators/rails/app/app_generator'
-          gen = Rails::Generators::AppGenerator.new ["rails"], { with_dispatchers: true, api: !!Rails.application.config.api_only },
-            destination_root: Rails.root
+          gen = Rails::Generators::AppGenerator.new ["rails"],
+                                                    { api: !!Rails.application.config.api_only },
+                                                    destination_root: Rails.root
           File.exist?(Rails.root.join("config", "application.rb")) ?
```

Comment:
Updated the code to not use array argument for `--api` as we don't have consensus yet about PR https://github.com/rails/rails/pull/22790. This PR just removes passing `with_dispatchers` option now.